cas-server-support-mongo/src/test/resources/cas.properties

```diff
@@ -0,0 +1,6 @@
+cas.authn.mongo.collection.name=users
+cas.authn.mongo.db.host=mongodb://casuser:Mellon@ds061954.mongolab.com:61954/jasigcas
```

Comment:
I could enable this test for mongolab, but I am concerned about performance. Guess we could give it a try. Need to make sure the db is cleaned before running tests because we only have about 500MB of free data, and overtime it might grow and fail tests.